### PR TITLE
Contour for 1.0

### PR DESCRIPTION
**Pull request by cw-sakamoto**
- Modifed chart for contour 1.0

### CODE REVIEW

File: contour/templates/contour-clusterrole.yaml [hunk 1]
```diff
@@ -43,10 +43,30 @@ rules:
...
+  - list
+  - watch
+  - put
+  - post
+  - patch
+- apiGroups:
+  - "projectcontour.io"
+  resources:
+  - "httpproxies"
+  - "tlscertificatedelegations"
```
Comment (cw-ozaki) [inline contour/templates/contour-clusterrole.yaml]: @cw-sakamoto 
https://github.com/chatwork/charts/pull/99/files#diff-71d473e431fde88db8c4b6c7d50c72b3R48-R69
How did you decide on these parameters?

If it is a Contur standard, please comment the documentation link.
Or comment why this parameter.

**Comment by cw-ozaki** [comment]
@cw-sakamoto 
I am very happy if you add the changes to the description.
・bump version to `1.0.0-rc.2`
・change organization from `heptio` to `projectcontour`
・reasons for other changes

**Comment by cw-ozaki** [comment]
@cw-sakamoto 
> WIP: push contour files

I don't see this commit message as good.
Because the master should not commit WIP.

**Comment by cw-ozaki** [comment]
Commit message is great! !

File: contour/templates/contour-clusterrole.yaml (hunk 1 above)
Comment (cw-sakamoto) [reply contour/templates/contour-clusterrole.yaml]: I confirmed clusterrole/role in 
https://github.com/projectcontour/contour/blob/master/examples/contour/02-rbac.yaml

I added comment at https://github.com/chatwork/charts/pull/99/commits/890dde9691b7c61a981ee26f25e8762ceaf2cd76

**Pull request merged**